include/bitcoin/bitcoin/chain/chain_state.hpp

```diff
@@ -207,14 +210,16 @@ class BC_API chain_state
     // easy blocks
 
     //TODO(bitprim):
-    
+
+#ifdef BITPRIM_CURRENCY_BCH
     static uint32_t cash_difficulty_adjustment(data const& values);
+    static uint32_t work_required_adjust_cash(const data& values);
+#endif //BITPRIM_CURRENCY_BCH
```

Comment:
Wouldn't it be cleaner if all the cash stuff were in a separate file? (not necessarily another class, just another file)